ui/rdb/types.d.ts

```diff
@@ -0,0 +1,4 @@
+declare var acquireVsCodeApi: Function;
+declare var vscode;
+declare var requestArgsSchema;
+declare var odcCommands;
```

Comment:
Can you make all of these variables `const` instead? Also, please add types for these.